Assets/MixedRealityToolkit/_Core/Definitions/Utilities/SixDof.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities
+{
+    [Serializable]
+    public struct SixDof : IEqualityComparer
```

Comment:
I need to pull a few of these changes out and make them their item PR.

Sorry about this.